CHANGELOG.md

```diff
@@ -35,6 +35,7 @@
 ### Improvements
 
 - **Azure Queue:** Don't call Azure queue GetProperties API unnecessarily ([#2613](https://github.com/kedacore/keda/pull/2613))
+- **Datadog Scaler:** Validate query to contains `{` to prevent panic on invalid query ([#2629](https://github.com/kedacore/keda/pull/2629))
```

Comment:
```suggestion
- **Datadog Scaler:** Validate query to contain `{` to prevent panic on invalid query ([#2629](https://github.com/kedacore/keda/pull/2629))
```
Typo

Reply:
nevermind, I'll just fix it locally and re-push :)